docs/ref/contrib/admin/filters.txt

```diff
@@ -0,0 +1,188 @@
+.. _modeladmin-list-filters:
+
+=======================
+ModelAdmin List Filters
+=======================
+
+.. currentmodule:: django.contrib.admin
+
+Model admin classes can define list filters that appear in the right sidebar of
+the change list page of the admin, as illustrated in the following screenshot:
+
+.. image:: _images/list_filter.png
+
+To activate per-field filtering, set :attr:`ModelAdmin.list_filter` to a list
+or tuple of elements, where each element is one of the following types:
+
+#. A field name
+#. A subclass of ``django.contrib.admin.SimpleListFilter``
+#. A tuple containing a field name and a subclass of
+   ``django.contrib.admin.FieldListFilter``
+
+See the examples below for discussion of each of these options for defining
+``list_filter``.
+
+Using a field name
+==================
+
+The simplest option is to specify the required field names.
```

Comment:
Do we know that by the time people read this bit, they'll have an understanding of how the `ModelAdmin` maps against a `Model`, or is it necessary to clarify that they're `Model` field names?